CHANGELOG.md

```diff
@@ -37,7 +37,6 @@ Releasing is documented in RELEASE.md
 
 ### Fixed
 - fix outdated dockerfile base images and add a more robust native golang base image ([#2027](https://github.com/GIScience/openrouteservice/pull/2027))
-- fix github action build default builder image failing ([#2032](https://github.com/GIScience/openrouteservice/issues/2032)) 
```

Comment:
I believe you should be adding an entry rather than deleting an existing one.. 😅